lib/ansible/module_utils/network/common/facts/facts.py

```diff
@@ -31,18 +31,21 @@ def __init__(self, module):
         if not self._gather_network_resources:
             self._gather_network_resources = ['!all']
 
-    def gen_runable(self, subsets, valid_subsets):
+    def gen_runable(self, subsets, valid_subsets, resource_facts=False):
         """ Generate the runable subset
 
         :param module: The module instance
         :param subsets: The provided subsets
         :param valid_subsets: The valid subsets
+        :param resource_facts: A boolean flag
```

Comment:
Just a note... "A boolean flag" probably isn't a very good piece of documentation ;-)